### PR TITLE
Jetpack 7.2: changelog

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* Changelog for the next release

#### Testing instructions:

* None

#### Proposed changelog entry for your changes:

* None

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -99,80 +99,37 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
-* Plans: clarify upgrade prompts in the Jetpack dashboard's Plans page.
-* Publicize: remove the Google+ interface as the Social Network is now deprecated.
-* Related Posts: update block to allow for up to 6 related posts.
-* Social Menus & Icons: add Discord Support.
-* Support: add additional tests to check when Jetpack isn't working as expected and ensures all current debugging platforms use the same testing list.
-* Simple Payments: add generic currency fallback symbol.
-* Sync: further performance improvements in PHP 7+ environments.
-* Woocommerce Analytics: include product type with analytics data.
+* Admin Dashboard: Various text and design improvements to make your administration experience all the better, including:
+** My Plan page is now more useful and actionable for free plans.
```
Comment (jeherve) [inline readme.txt]: You have an extra `*` in here.

File: readme.txt [hunk 2]
```diff
@@ -99,80 +99,37 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
 
-* Block Editor: this release introduces 6 new blocks:
-	- the Ads block allows you to insert different ads from [our WordAds program](https://jetpack.com/support/ads/) within your posts and pages.
-	- the Mailchimp block allows your readers to easily subscribe to your Mailchimp newsletter.
-	- the Video block supports VideoPress videos if you've purchased our Premium or Professional plan.
-	- the Slideshow block allows you to insert beautiful slideshows in your posts and pages.
-	- The Business Hours blocks is useful for companies who want to display their business's Opening Hours on their site.
-	- The Contact Info block is useful for any business who may want to display useful information on a post or page.
+* Adds a Repeat Visitor block that controls block visibility based on how often a visitor has viewed the page.
+* Selectively disable Ads blocks for mobile devices.
```
Comment (jeherve) [inline readme.txt]: I would expand on that to make it clear we are adding a new option.

**Review by jeherve — COMMENTED** [review]
What do you think about sticking to a consistent format for each changelog line? In the past I would try to have a title / module / feature, a colon, and then a verb using the present tense.

So something like this:

```
* Shortcodes: Update
* Sitemaps: Added
```

would turn into:

```
* Shortcodes: update
* Sitemaps: add
```

What do you think?

File: readme.txt (hunk 1 above)
Comment (kraftbj) [reply readme.txt]: I was thinking of having it be a nested list if there was enough Admin Dashboard related things to note. There my not be and would condense that down.

File: readme.txt [hunk 3]
```diff
@@ -99,80 +99,37 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
 **Improved compatibility**
 
-* General: replace all .dev TLD references by .test as the .dev TLD will soon become available for registration.
-* PHP 7.3: introduce automated testing for PHP 7.3.
-* Site Accelerator: ensure compatibility with the AMP plugin.
-* Twenty Nineteen: fix Top Posts and Pages Widget image list margins.
-* WordPress 5.1 Compatibility: update usage of `wp_schedule_single_event` to match changes in WordPress.
+* General: Jetpack now requires WordPress 5.0! To celebrate, we cleaned out some compatibility code that supported older versions. We know how to party.
+* Shortcodes: Update the Ustream shortcode to use the HTML5 player for a better experience on all browsers.
+* Sitemaps: Added thumbnails to video sitemaps to improve compatability with by Google Search Console. Props Adam Heckler!
```
[the remark this answers is not in the document]
Comment (kraftbj) [reply readme.txt]: I'm open to it for non-a11n developers or non-developer a12s. Adding it while I was working on a commit and see what y'all thought about it.

File: to-test.md [hunk 4]
```diff
@@ -1,96 +1,16 @@
-## 7.1
+## 7.2
 
-### Block Editor
+- Return Visitor block. Add the new Return Visitor block in the block editor and try various options. The visitor count is stored in as the cookie value for `jp-visit-counter`.
```
[the remark this answers is not in the document]
Comment (kraftbj) [reply to-test.md]: Yeah, for the to-test items I wanted to expand out both of them. More of wanting these to be placeholders for the immediate moment.

File: readme.txt (hunk 2 above)
Comment (kraftbj) [reply readme.txt]: 👍

**Comment by kraftbj** [comment]
```
* Shortcodes: Update
* Sitemaps: Added

would turn into:

* Shortcodes: update
* Sitemaps: add

What do you think?```

I'm fine with that. Want me to stick to the lower-case after the colon?

**Comment by jeherve** [comment]
> Want me to stick to the lower-case after the colon?

Up to you. I think the most important is to remain consistent across all lines in the existing changelog.

File: readme.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (jeherve) [reply readme.txt]: Why not? I would only suggest removing the exclamation marks.

File: readme.txt (hunk 1 above)
Comment (jeherve) [reply readme.txt]: Oh I see, that makes sense now. I would condense it into one changelog entry, to be honest. I don't think the details are going to be very interesting for most people.

File: readme.txt (hunk 2 above)
Comment (kraftbj) [reply readme.txt]: fb97ab5ae

**Pull request merged**